src/api-state/sync/index.js

```diff
@@ -8,18 +8,40 @@ import 'rxjs/add/operator/catch';
 import 'rxjs/add/operator/map';
 import 'rxjs/add/operator/mergeMap';
 import 'rxjs/add/operator/takeUntil';
-
 import { combineEpics } from 'redux-observable';
-import * as api from '../actions/apiActionCreators';
+
+import { actions as clickActions } from '../../plugins/tracking/util/clickState'; // mwp-tracking-plugin/util/clickState
+import { LOCATION_CHANGE, SERVER_RENDER } from '../../router'; // mwp-router
+import { getRouteResolver, getMatchedQueries } from '../../router/util'; // mwp-router/util
+
+import * as api from './apiActionCreators';
 import {
 	apiSuccess, // DEPRECATED
 	apiError, // DEPRECATED
-	LOCATION_CHANGE,
-	SERVER_RENDER,
-} from '../actions/syncActionCreators';
-import { actions as clickActions } from '../plugins/tracking/util/clickState';
-import { getRouteResolver, getMatchedQueries } from '../router/util';
-import { getDeprecatedSuccessPayload } from '../util/fetchUtils';
+} from './syncActionCreators';
+
+/**
+ * @deprecated
+ */
+export function getDeprecatedSuccessPayload(successes, errors) {
```

Comment:
I moved this function over from `fetchUtils` because it's only used here and `fetchUtils` doesn't need to be a dependency